global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.1.504"
```

Comment:
Why does this not work with later versions of the SDK?

Reply:
Some of the NuGet dependencies were getting resolved differently and it wouldn't complete a build. You can see the long tail of failed AppVeyor builds for examples. Even a trivial no-op commit will trigger a build failure in the current state of the repository.

After this change is merged, we can go back and work to update the SDK version if we want. This at least gets us to a state where builds work though.